docs/loaders-context.rst

```diff
@@ -0,0 +1,42 @@
+.. _loaders-context:
+
+Item Loader Context
+===================
```

Comment:
This section should mention what data is available in context by default; as I understand, it is `selector` object. That's fine to fix it in this PR, or in another PR; if you want to move it out of this PR, could you please raise a github issue for this?

Reply:
Changed the wording a bit

> The Item Loader Context is a mechanism that allows to change the input/ouput processors behavior.
It's just a ``dict`` of arbitrary key/values which is shared among all processors.
By default, the context is set to the ``selector`` and any other `keyword arguments`
sent to the Loaders's ``__init__``.
The context can be passed when declaring, instantiating or using Item Loader.